File: Android/MyApplication/app/src/main/java/com/willy/myapplication/dto/InvestPlanRowDto.java
package com.willy.myapplication.dto;

import java.math.BigDecimal;

public class InvestPlanRowDto {
    private BigDecimal amt;
    private BigDecimal index;

    public InvestPlanRowDto(BigDecimal amt, BigDecimal index) {
        this.amt = amt;
        this.index = index;
    }

    public BigDecimal getAmt() {
        return amt;
    }

    public void setAmt(BigDecimal amt) {
        this.amt = amt;
    }

    public BigDecimal getIndex() {
        return index;
    }

    public void setIndex(BigDecimal index) {
        this.index = index;
    }

    @Override
    public String toString() {
        return "InvestPlanRowDto{" +
                "amt=" + amt +
                ", index=" + index +
                '}';
    }
}

File: Flutter/sias/lib/dto/InvestPlanRowDto.dart
import 'package:decimal/decimal.dart';

class InvestPlanRowDto {
  Decimal? _amt;
  Decimal? _index;

  InvestPlanRowDto(amt, index) {
    _amt = amt;
    _index = index;
  }

  Decimal? get amt => _amt;
  set amt(Decimal? value) => _amt = value;

  Decimal? get index => _index;
  set index(Decimal? value) => _index = value;
}